package.json

```diff
@@ -35,15 +32,16 @@
     "@turf/turf": "^6.3.0",
     "applicationinsights": "^1.8.10",
     "blipp": "^4.0.2",
-    "ffc-messaging": "^2.1.3",
+    "ffc-messaging": "^2.3.0",
```

Comment:
This repo doesn't use `ffc-messaging` so better to just remove this rather than upgrade it.